src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobDispatcher.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import org.slf4j.Logger;
+
+import java.io.ByteArrayInputStream;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class BlobDispatcher {
+
+    private static final Logger logger = getLogger(BlobDispatcher.class);
+
+    private final BlobServiceClient storageClient;
+
+    public BlobDispatcher(BlobServiceClient storageClient) {
+        this.storageClient = storageClient;
+    }
+
+    public void dispatch(String blobName, byte[] blobContents, String destinationContainer) {
+        logger.info("Uploading {} to {} container", blobName, destinationContainer);
+
+        try {
+            getContainerClient(destinationContainer)
+                .getBlobClient(blobName)
+                .getBlockBlobClient()
+                .upload(
+                    new ByteArrayInputStream(blobContents),
+                    blobContents.length,
+                    false // overwrite flag
```

Comment:
If we are not overwriting then it will create snapshots ?

We are not handling snapshots so may be overwrite it for now ?

Reply:
i can remove the flag addition